ansible_collections/arista/avd/plugins/modules/yaml_templates_to_facts.py

```diff
@@ -77,12 +81,45 @@
       Autodetects data format based on file suffix. '.yml', '.yaml' -> YAML, default -> JSON
     required: false
     type: str
+  mode:
+    description: File mode for dest file.
```

Comment:
I a not quite sure what `file mode` is when I read it like this maybe add the example format (I know that this is the name ansible uses for its file module but still..)

Reply:
it is already set under `vars/main.yml`

Reply:
fixed